rotkehlchen/db/ethtx.py

```diff
@@ -245,6 +245,17 @@ def get_transaction_hashes_no_receipt(
 
         return hashes
 
+    def get_all_transaction_hashes(self) -> List[EVMTxHash]:
+        cursor = self.db.conn.cursor()
+        cursor_result = cursor.execute('SELECT DISTINCT tx_hash FROM ethereum_transactions')
```

Comment:
DISTINCT is not needed here. It's the primary key
```suggestion
        cursor_result = cursor.execute('SELECT tx_hash FROM ethereum_transactions')
```

Reply:
Okay.